hca-dcp/submit.wdl

```diff
@@ -23,7 +24,9 @@ task get_metadata {
       "$(cut -f3 $creds)/api/workflows/v1/$(cat workflow_id.txt)/metadata" > metadata.json
   >>>
   runtime {
-    docker: "gcr.io/broad-dsde-mint-dev/cromwell-metadata:0.1.0"
+    docker: if runtime_environment == "dev" then "gcr.io/broad-dsde-mint-dev/cromwell-metadata:0.1.0"
+            else if runtime_environment == "staging" then "gcr.io/broad-dsde-mint-staging/cromwell-metadata:0.1.0"
+            else "We cannot recognize the environment: " + runtime_environment
```

Comment:
@dshiga Recommended me that I could simplify this if-else statement to a better style such as: `"gcr.io/broad-dsde-mint-" + runtime_environment + "/cromwell-metadata:0.1.0"`

Reply:
Yes, the benefit is that it's much simpler, so less chance of introducing a bug, and it's easier for other developers to read and understand. If someone passes an environment other than dev or staging, it will break this task and should be pretty straightforward to debug.